### DISCUSSION TITLE
Split the output file by each tools

### DISCUSSION THREAD

**Discussion by yukleides** [TimPaterson/Fusion360-Batch-Post#46] [Ideas]
Hi

It would be nice to be able to split the output file by each tools.
In case of some trouble, it can be resumed from the middle of the operation.

Best regards

**Comment by TimPaterson** [comment]
Organize around one file per setup.  Duplicate the setup if you want two files with the same setup.

**Reply by yukleides** [reply]
Thank you for the reply.
I know duplicate the setup can split the output file.
But I prefer to automatically do it.

By the way, I found a behavior that seems to be a bug.
The Z is offset when crossing over the machining process. At that time, the value of A-axis, which is not used, also changes.
What information should I share for bug analysis?

**Reply by yukleides** [reply]
The post processor is mach3mill.cps

**Comment by TimPaterson** [comment]
I don't know what 
`The Z is offset when crossing over the machining process.`
means. And PostProcessAll never does anything with the A axis.  You can uncheck "restore rapid moves" to see what the original G-code looked like, but there is also a comment with the original code anytime it makes a change.

**Reply by yukleides** [reply]
I'm sorry for late reply.
I checked the G-codes and could not find any code that would change the A-axis.
But, in fact, A-axis changes on Mach3 when crossing over the machining process.

Just my idea ....is to split the output files by each machining process.
